### PR TITLE
Add PT parser

### PR DESCRIPTION
**Pull request by guicoelho**
* Add parser for Portugal (PT) using data from http://www.centrodeinformacao.ren.pt/EN/InformacaoExploracao/Pages/EstatisticaDiaria.aspx

Note: Data is refreshed daily.. not as "real-time" as other sources but still thought it's better than having a grey country.

### CODE REVIEW

File: feeder/requirements.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ pymongo==3.2.2
 requests==2.10.0
 schedule==0.3.2
 xlrd==1.0.0
+beautifulsoup4==4.5.1
```
Comment (corradio) [inline feeder/requirements.txt]: can you add in alphabetical order?

File: feeder/parsers/PT.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import requests, dateutil, arrow
+from bs4 import BeautifulSoup
+
+COUNTRY_CODE = 'PT'
+
+def fetch_PT():
+
+	r = requests.get('http://www.centrodeinformacao.ren.pt/EN/InformacaoExploracao/Pages/EstatisticaDiaria.aspx')
+	soup = BeautifulSoup(r.text, 'html.parser')
```
Comment (corradio) [inline feeder/parsers/PT.py]: can you quickly explain what this library is for? Just curious :)

File: feeder/parsers/PT.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		    'wind': daily_values[9],
+		    'solar': daily_values[10],
+		    'hydro': daily_values[0] + daily_values[7], # There are 2 different production regimes
+		    'coal': daily_values[1] + daily_values[8] # There are 2 different production regimes
+		},
+		'consumption': {
+		    'unknown': daily_values[13]
+		},
+		'exchange':{
+			'ES': daily_values[3] - daily_values[4]
```
Comment (corradio) [inline feeder/parsers/PT.py]: 👍

**Comment by corradio** [comment]
You should also update those files:
- https://github.com/corradio/electricitymap/blob/master/feeder/feeder.py
- https://github.com/corradio/electricitymap/blob/master/api/static/app/exchangeconfig.js
- https://github.com/corradio/electricitymap/blob/master/api/static/app/countryconfig.js

File: feeder/parsers/PT.py (hunk 2 above)
Comment (guicoelho) [inline feeder/parsers/PT.py]: It's an html parser: text -> tree! AFAIK it's the most used html parser for Python.

File: feeder/requirements.txt (hunk 1 above)
Comment (guicoelho) [inline feeder/requirements.txt]: 👍

**Pull request merged**